openid-sharedsignals-framework-1_0.md

```diff
@@ -832,24 +846,10 @@ the Event Transmitter responds with a "201 Created" response containing a
 The HTTP POST request MAY contain the Receiver-Supplied values of the Stream
 Configuration ({{stream-config}}) object:
 
-events_requested
-
-> **Receiver-Supplied**, An array of URIs identifying the set of events that
-  the Receiver requested. A Receiver SHOULD request only the events that it
-  understands and it can act on. This is configurable by the Receiver.
-
-delivery
-
-> **Receiver-Supplied**, A JSON object containing a set of name/value pairs
-  specifying configuration parameters for the SET delivery method. The actual
-  delivery method is identified by the special key "method" with the value
-  being a URI as defined in {{delivery-meta}}.
-
-format
-
-> **Receiver-Supplied**, The Subject Identifier Format that the Receiver wants
-  for the events. If not set then the Transmitter might decide to use a type
-  that discloses more information than necessary.
+* `events_requested`
+* `delivery` : Note that in the case of the POLL method, the `url` value is
```

Comment:
Since Receivers MAY include these values, we should say that if the `delivery` value is not included in the stream configuration request, a Transmitter MUST respond with polling as the delivery method.

Reply:
added